phpcs.xml.dist

```diff
@@ -8,7 +8,7 @@
 	<rule ref="WordPress-Docs"/>
 	<rule ref="WordPress-Extra" />
 	<rule ref="WordPress.WP.I18n"/>
-	<config name="text_domain" value="performance-lab,default"/>
+	<config name="text_domain" value="performance-lab,default,auto-sizes"/>
```

Comment:
We may want to have separate `phpcs.xml.dist` files in each plugin rather than putting all the text domains in the same file. This would help us catch situations where someone accidentally used the wrong text domain from one plugin in another because of a sloppy copy/paste. Not a blocker for now though.

Reply:
Can we open issue for that and work on follow-up PR?

Reply:
Open a separate issue for further work: https://github.com/WordPress/performance/issues/997